calico/datadog_checks/calico/check.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+class CalicoCheck(OpenMetricsBaseCheck):
+    def __init__(self, name, init_config, instances=None):
+        METRICS_MAP = {'felix_active_local_endpoints': 'felix_active_local_endpoints'}
+
+        super(CalicoCheck, self).__init__(
+            name,
+            init_config,
+            instances,
+            default_instances={
+                'calico-felix': {
+                    'prometheus_url': 'http://localhost:9091/metrics',
+                    'namespace': 'kube-system',
+                    'metrics': [METRICS_MAP],
+                }
+            },
+            default_namespace='kube-system',
+        )
+
+    def check(self, instance):
+        endpoint = instance.get('prometheus_url')
+        if endpoint is None:
+            raise ConfigurationError("Unable to find prometheus_url in config file.")
+        config = self.create_scraper_configuration(instance)
+        self.process(config)
+        pass
```

Comment:
```suggestion
```
You can delegate on the parent class https://github.com/DataDog/integrations-core/blob/cfd04ffcaed83b58f501d54e575371228ed727ed/datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/base.py#L28

Validation of parameters is done by the config models. You can run `ddev validate models -s calico` to generate them

Reply:
I'm having issues when generating config models and running `ddev test calico`, I couldn't find any solution yet : 
`E   ImportError: cannot import name 'validators' from partially initialized module 'datadog_checks.calico.config_models' (most likely due to a circular import)` (from `config_models/shared.py`)